src/boot/bootctl.c

```diff
@@ -1647,6 +1653,22 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_arch_all = true;
                         break;
 
+                case ARG_EFI_LABEL:
+                        if (isempty(optarg) || !string_is_safe(optarg)) {
```

Comment:
maybe also utf8 validate the string, i.e. use the utf8_is_valid() helper

Reply:
still missing

Reply:
Not sure at which point this code change went missing... now its there